### ISSUE TITLE
Missing Liquid blinding key and not able to access L-Btc balance

### ISSUE THREAD

**Issue by farukterzioglu** [dgarage/NBXplorer#304]
I received a transaction to a confidential address generated via NBXplorer. But that amount isn't reflected to the wallet. Balance didn't change even I can see the transaction in wallet. 

When I call `elements-cli dumpblindingkey [CONFIDENTIAL_ADDRESS]` it returns null, so probably the blinding key wasn't imported at the correct moment (maybe a temporary rpc error)

I resolved it by creating the blinding key of that confidential address and by calling `elements-cli importblindingkey [CONFIDENTIAL_ADDRESS] [BLINDING_KEY]`

The code I used to create blinding key;

```
var mainNetworkProvider = new NBXplorerNetworkProvider(NetworkType.Mainnet);
network = mainNetworkProvider.GetFromCryptoCode("LBTC");

var strategy = new DerivationStrategyFactory(network.NBitcoinNetwork).Parse("xpub....");

var blindingKey = NBXplorerNetworkProvider.LiquidNBXplorerNetwork.GenerateBlindingKey(strategy, new KeyPath("0/1")); // for address at 0/1
var hex = blindingKey.ToHex();
```

I resolved my issue but someone can see this issue and take this as a reference.
Or maybe we can deep dive for the reason and prevent for further occurrences, otherwise this issue can be closed.

@Kukks helped me with this in https://chat.btcpayserver.org/btcpayserver/channels/liquid

**Comment by NicolasDorier** [comment]
Yeah that's probably why. We should fix it by trying again if it fails.
